bin/compile

```diff
@@ -157,14 +157,13 @@ cache_build() {
   if ! ${NODE_MODULES_CACHE:-true}; then
     echo "Skipping cache save (disabled by config)"
   elif [ "$cache_directories" == "" ]; then
-    echo "Saving 3 cacheDirectories (default):"
+    echo "Saving 2 cacheDirectories (default):"
     save_cache_directories "$BUILD_DIR" "$CACHE_DIR" "$DEFAULT_CACHE"
   else
     echo "Saving $(echo $cache_directories | wc -w | xargs) cacheDirectories (package.json):"
     save_cache_directories "$BUILD_DIR" "$CACHE_DIR" $cache_directories
   fi
   save_signature
-  remove_caches_from_slug "$BUILD_DIR"
```

Comment:
do we need to leave this in here to clean up caches that started caching this already if it's detected?